src/allencell_ml_segmenter/core/file_input_widget.py

```diff
@@ -50,14 +50,14 @@ def __init__(
         self,
         model: FileInputModel,
         viewer: IViewer,
-        service: ModelFileService,
+        service: Optional[ModelFileService],
```

Comment:
I think exposing a second constructor for the subclasses would be a nice way to get around making `service` Optional.